bin/all_sky_search/pycbc_template_kde_calc

```diff
@@ -0,0 +1,147 @@
+#3dimensional_kde 
+#python3 pycbc_template_kde_calc.py --template-file /home/praveen.kumar/focused_search/kde/sbbh2.hdf --signal-file /home/praveen.kumar/focused_search/kde/3OGC_PEtable.txt_ifar_gt0.5.csv     --make-signal-kde    --verbose
+
+import numpy as np
+import operator
+import pycbc
```

Comment:
I don't think you need to import everything from pycbc ...  I think `from pycbc import init_logging` would be preferable